### ISSUE TITLE
GI Bill Statement of Benefits - Content updates needed

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#79971]
## Background
There are several areas that need content attention on the statement of benefits tool:

1. The page need a content review; it seems to have some extra words and awkward sentences in the intro that don't align with our voice and tone, and there are a couple of other spots that feel off.
2. We should be sending people to our payment history tool instead of eBenefits (first horizontal red line in screenshot)
3. In the bottom half, the headings look wonky to me. Maybe not a problem but should have a content review and potentially check with VADS about the use of the summary box. (vertical red highlight)
4. We’re missing TTY info with the phone number at the bottom (last red highlight). This is an accessibility consideration.

<img width="433" alt="post-911-gibill" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/55992116/95410648-c05d-4596-a245-3ba3dc11825e">
